tests/functional/spec/components/checkbox/mutually_exclusive/mutually_exclusive_dropdown.spec.js

```diff
@@ -45,6 +45,7 @@ describe('Component: Mutually Exclusive Dropdown With Single Checkbox Override',
 
         // When
         .selectByValue(DropdownPage.dropdown(), 'Liverpool')
+        .click(DropdownPage.dropdownExclusiveIPreferNotToSayLabel())
```

Comment:
By clicking the label you may be effectively clicking the checkbox so this isn't a safe way of fixing this. I think `selectByValue` may need to be modified to trigger a change event.